index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=EB+Garamond&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/7acb0c92dc.js" crossorigin="anonymous"></script>
+    <meta property ="og:title" content="Hannahs Portfolio"/>
+    <meta property ="og:description" content="The portfolio page of Hannah Ek code coach at Technigo, lorem ipsum set dolor amet"/>
+    <meta property ="og:image" content="https://placehold.co/400"/>
+    <title>Hannah's Portfolio</title>
+</head>
+<body>
+<head>
+    <!--Hela sektionen-->
+   
+    <section class="main">
+    <div class="container">
+    <img src="./images/hero-image.jpg">
+</div>
+<div class="container2">
+    <div class="profile-image">
+        
+        <img src="./images/333609906_753068772790998_6707688347266238785_n.jpg" alt="Profile picture of Hannah Ek">
+    </div>
+    <!--Profilbild-->
+    <div class="profile-header">
+  
+<!--Info texten-->
+<div class="info">
+<h>Hannah Ek</h>
+<h1>Frontend Developer</h1></div>
+<p>Hannah is an exceptional developer known for her innovative solutions and exceptional coding abilities. She creates user-friendly applications and solves complex issues with ease. 
+    Her drive for </p>
+</div>
+    <!--Länkar till social media-->
+<section class="social-media-links">
+    <button class="social-media-button">
```

Comment:
I noticed these buttons aren't functional. I solved it like this:
```suggestion
    <a class="" href="https://" title="" target="_blank">
                <svg>
                </svg>   
    </a>
```
When you get the hang of svg-icons in code form, they are easier to resize etc.